### PR TITLE
feat(icons): add enterprise file storage icon

### PR DESCRIPTION
**Pull request by marie-j**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: packages/base/core/src/fonts/oui-icons/oui-icons.html [hunk 1]
```diff
@@ -682,6 +682,15 @@ <h1>oui-icons</h1>
         </div>
 
 
+        <div class="preview">
+            <span class="inner">
+                <span class=" oui-icon-enterprise-file-storage_concept"></span>
+            </span>
+            <br>
+            <span class='label'>enterprise-file-storage_concept</span>
```
Comment (JeremyDec) [inline packages/base/core/src/fonts/oui-icons/oui-icons.html:690]: **nitpick**
```suggestion
            <span class="label">enterprise-file-storage_concept</span>
```

Comment (marie-j) [reply packages/base/core/src/fonts/oui-icons/oui-icons.html:690]: File has been autogenerated

**Review by JeremyDec — DISMISSED** [review]

File: packages/base/core/src/svg/icons/concept/enterprise-file-storage_concept.svg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<svg id="Calque_1" data-name="Calque 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><defs><style>.cls-1 { fill: #113f6d; }
```
Comment (antleblanc) [inline packages/base/core/src/svg/icons/concept/enterprise-file-storage_concept.svg:1]: **thought:** Both `id=` attributes and `data-name` attribute seems useless. Isn't it?

Comment (marie-j) [reply packages/base/core/src/svg/icons/concept/enterprise-file-storage_concept.svg:1]: You are right, we used the SVG that was provided

**Review by JeremyDec — APPROVED** [review]

**Review by lizardK — APPROVED** [review]

**Review by zakariasahmane — APPROVED** [review]

**Review by cbourgois — APPROVED** [review]

**Pull request merged**